### ISSUE TITLE
p5.js sketch breaks when I save it

### ISSUE THREAD

**Issue by chris-relaxing** [fathominfo/processing-p5js-mode#18]
If I start up a new sketch in p5.js mode and paste in this code, it works fine. I click run and it shows the output in my browser window with no errors (correctly showing a square spinning on the Y axis)

```
let angle = 0;

function setup() {
createCanvas(400, 300, WEBGL);
}

function draw() {
background(175);
rectMode(CENTER);
//noStroke();
//fill(0, 0, 255);
//rotateZ(angle);
rotateY(angle);
rect(0, 0, 150, 150);
angle += 0.03;

}
```

However, if I then save this file, it no longer works when I try to run it. I get an error message:
SyntaxError: Expected ; but found angle
And this generally highlights the first line in the code.

I am using Processing IDE 3.3.6. Windows 10. Firefox for output.

Processing environment.
![image](https://user-images.githubusercontent.com/30215910/34268954-419e542c-e648-11e7-846f-64328df1968c.png)

p5.js mode:
![image](https://user-images.githubusercontent.com/30215910/34268956-45964f30-e648-11e7-8d9e-9873b1156e3a.png)

error message:
![image](https://user-images.githubusercontent.com/30215910/34268950-3c8e70d4-e648-11e7-9ed5-db9a94e65cb9.png)

index.html:
![image](https://user-images.githubusercontent.com/30215910/34268948-38e41272-e648-11e7-98a0-60c6c0772536.png)

**Comment by schellenberg** [comment]
I was able to replicate this error, as well as find a *partial* solution. I was guessing the error was occurring due to some kind of syntax checking that p5js-mode was using. On a hunch, I changed the ``let`` to ``var``, saved the file again, and it executed just fine. 

Poking around the source of p5js-mode, I'm wondering if it is the ``public HtmlTokenMarker(boolean js)`` function in the ``HTMLTokenMarker.java`` file. I can see ``var`` in there, but the more recent ``let`` and ``const`` ES6 style variable declarations are not. Perhaps it is as simple as adding those?
